docs/docs/feature-library/teams-view-filters.md

```diff
@@ -66,17 +66,6 @@ To use the sample features, most of the filters can simply be toggled on in the
           "Purchasing",
```

Comment:
food for thought, granted department isnt actually required by any other features but i feel like it intuitively belongs in the common area of configuration now.  What do you think?

Reply:
I don't disagree! I will move this.